src/Makefile

```diff
@@ -65,3 +68,7 @@ check-physlr-makebf: all
 check-physlr-overlap: all
 	./physlr-overlap -t4 -n1 data/tiny.overlap.input.tsv | sort | diff -q - data/tiny.physlr.overlap.n1.sorted.tsv
 	./physlr-overlap -t4 -n2 data/tiny.overlap.input.tsv | sort | diff -q - data/tiny.physlr.overlap.n2.sorted.tsv
+
+install: physlr-indexlr physlr-filter-barcodes physlr-overlap physlr-filter-bxmx physlr-makebf
+	mkdir -p $(DESTDIR)$(PREFIX)/bin
```

Comment:
```suggestion
	install -d $(DESTDIR)$(PREFIX)/bin
```

Reply:
i used `install $^ -t $(DESTDIR)$(PREFIX)/bin` instead. Is that alright?

Reply:
Nevermind, the above command doesn't seem to make the directory if it doesn't exist